src/addresses.ts

```diff
@@ -13,228 +11,21 @@ type ChainAddresses = {
-}
-
-const ZORA_ADDRESSES: ChainAddresses = {
-  v3CoreFactoryAddress: '0x7145F8aeef1f6510E92164038E1B6F8cB2c42Cbb',
-  multicallAddress: '0xA51c76bEE6746cB487a7e9312E43e2b8f4A37C15',
-  quoterAddress: '0x11867e1b3348F3ce4FcC170BC5af3d23E07E64Df',
-  v3MigratorAddress: '0x048352d8dCF13686982C799da63fA6426a9D0b60',
-  nonfungiblePositionManagerAddress: '0xbC91e8DfA3fF18De43853372A3d7dfe585137D78',
-  tickLensAddress: '0x209AAda09D74Ad3B8D0E92910Eaf85D2357e3044',
-  swapRouter02Address: '0x7De04c96BE5159c3b5CeffC82aa176dc81281557'
-}
-
-const ZORA_SEPOLIA_ADDRESSES: ChainAddresses = {
-  v3CoreFactoryAddress: '0x4324A677D74764f46f33ED447964252441aA8Db6',
-  multicallAddress: '0xA1E7e3A69671C4494EC59Dbd442de930a93F911A',
-  quoterAddress: '0xC195976fEF0985886E37036E2DF62bF371E12Df0',
-  v3MigratorAddress: '0x65ef259b31bf1d977c37e9434658694267674897',
-  nonfungiblePositionManagerAddress: '0xB8458EaAe43292e3c1F7994EFd016bd653d23c20',
-  tickLensAddress: '0x23C0F71877a1Fc4e20A78018f9831365c85f3064'
-}
-
-const ROOTSTOCK_ADDRESSES: ChainAddresses = {
-  v3CoreFactoryAddress: '0xaF37EC98A00FD63689CF3060BF3B6784E00caD82',
-  multicallAddress: '0x996a9858cDfa45Ad68E47c9A30a7201E29c6a386',
-  quoterAddress: '0xb51727c996C68E60F598A923a5006853cd2fEB31',
-  v3MigratorAddress: '0x16678977CA4ec3DAD5efc7b15780295FE5f56162',
-  nonfungiblePositionManagerAddress: '0x9d9386c042F194B460Ec424a1e57ACDE25f5C4b1',
-  tickLensAddress: '0x55B9dF5bF68ADe972191a91980459f48ecA16afC',
-  swapRouter02Address: '0x0B14ff67f0014046b4b99057Aec4509640b3947A'
-}
-
-const BLAST_ADDRESSES: ChainAddresses = {
-  v3CoreFactoryAddress: '0x792edAdE80af5fC680d96a2eD80A44247D2Cf6Fd',
-  multicallAddress: '0xdC7f370de7631cE9e2c2e1DCDA6B3B5744Cf4705',
-  quoterAddress: '0x6Cdcd65e03c1CEc3730AeeCd45bc140D57A25C77',
-  v3MigratorAddress: '0x15CA7043CD84C5D21Ae76Ba0A1A967d42c40ecE0',
-  nonfungiblePositionManagerAddress: '0xB218e4f7cF0533d4696fDfC419A0023D33345F28',
-  tickLensAddress: '0x2E95185bCdD928a3e984B7e2D6560Ab1b17d7274',
-  swapRouter02Address: '0x549FEB8c9bd4c12Ad2AB27022dA12492aC452B66'
-}
+// Hemi addresses
+// @TODO: enable once we have these addresses for HEMI chain. ref: https://github.com/hemilabs/interface/issues/31
+// We would also need to add HEMI to SUPPORTED_CHAINS (chains.ts)
+/*const HEMI_ADDRESSES: ChainAddresses = {
+  multicallAddress: '0xf7364E230EA0e5CC41E6Cd4E2a30A26Ca1850b73',
+  nonfungiblePositionManagerAddress: '0x703D7618Fa53fB0C2b7f28fD60DD9b1fD9084cC1',
+  quoterAddress: '0x3899a6090c5C178dB8A1800DA39daD0D06EeEFBE',
+  swapRouter02Address: '0x70523D3F46c9858D3319c4bb02a236Dc6C449d39',
+  tickLensAddress: '0x70AB149e550690D55a46AA326211438c5D47B6D3',
+  v3CoreFactoryAddress: '0xc2f8037a31a4f12b07d20D3f382b1aA03c94519B',
+  v3MigratorAddress: '0x1B1ff6ad47a9fe6eF10b9A235a2B2C8d1CED743e'
+}*/
 
+// Hemi Sepolia addresses
 const HEMI_SEPOLIA_ADDRESSES: ChainAddresses = {
   multicallAddress: '0xf7364E230EA0e5CC41E6Cd4E2a30A26Ca1850b73',
```

Comment:
Isn't this one `0xcA11bde05977b3631167028862bE2a173976CA11`? Double-check the rest.

Reply:
I think this address comes from [here](https://github.com/BVM-priv/uniswap-deploy-v3/blob/57c16ce7535adbf502b1040522a951a8c56060f1/state.json#L3) which was the list that Rohit shared originally when starting to work on Uniswap on Hemi. I don't know which would be now cc @rokso @max-sanchez

Reply:
@gndelia @gabmontes The one I have in uni repo is multicall2 and the one Gabriel has is multicall3.

Reply:
For uniswap, you will be using multicall2